src/Components/Index.jsx

```diff
@@ -17,7 +17,7 @@ function Index() {
           firstH1={'Ready to start actually sticking to your goals?'}
           span={'Sign up for our free ebook on the most effective tatics for optimising your life:'}
```

Comment:
What is that?
Please put that in `SignUp` component
Use props only if you need to pass some variable or something to child component